manifests/e/EaglePointSoftware/PinnacleSeriesManagementUtility/2023.2.1.101/EaglePointSoftware.PinnacleSeriesManagementUtility.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: EaglePointSoftware.PinnacleSeriesManagementUtility
+PackageVersion: 2023.2.1.101
+PackageLocale: en-US
+Publisher: Eagle Point Software
+PublisherUrl: https://www.eaglepoint.com/
+PublisherSupportUrl: https://support.eaglepoint.com/
+PrivacyUrl: https://www.eaglepoint.com/privacy-policy/
+Author: Eagle Point Software Corporation
+PackageName: Pinnacle Series
+License: Proprietary
+Copyright: Copyright © 2007-2018, by Eagle Point Software Corporation. All rights reserved.
+ShortDescription: user administration, platform branding, content customization, course enrollments, and reporting
+Description: Platform Administrators and Managers require a "thin client application" (Windows Based) to be installed onto their workstations. The Pinnacle Series Management Utility is required for user administration, platform branding, content customization, course enrollments, and reporting.
+Moniker: Pinnacle
```

```suggestion
Moniker: pinnacle
```
A moniker is a special type of tag. It should be written in all lowercase.